crates/holochain_persistence_lmdb/src/eav/lmdb.rs

```diff
@@ -78,18 +81,17 @@ where
         // use a clever key naming scheme to speed up exact match queries on the entity
         let mut new_eav = eav.clone();
         let mut key = format!("{}::{}", new_eav.entity(), new_eav.index());
-
         // need to check there isn't a duplicate key though and if there is create a new EAVI which
         // will have a more recent timestamp
         while let Ok(Some(_)) = self.lmdb.store.get(&reader, key.clone()) {
             new_eav = EntityAttributeValueIndex::new(&eav.entity(), &eav.attribute(), &eav.value())
-                .unwrap();
+                .map_err(|_| StoreError::DataError(DataError::Empty))?;
```

Comment:
:+1: 

Was this error observed in a happ? Just curious.

Reply:
It wasnt, just thought it would be a good thing to throw an error in there so that it bubbles on to the happ